api/app/signals/apps/signals/models/expression.py

```diff
@@ -0,0 +1,60 @@
+from django.contrib.gis.db import models
+
+
+class ExpressionType(models.Model):
+    class Meta:
+        verbose_name = 'ExpressionsType'
+        verbose_name_plural = 'ExpressionType'
+
+    id = models.AutoField(primary_key=True)
+    name = models.CharField(max_length=255, unique=True)
+    description = models.TextField(max_length=3000)
+
+    def __str__(self):
+        return self.name
+
+
+class ExpressionContext(models.Model):
+    CTX_POINT = 'point'
+    CTX_STRING = 'str'
+    CTX_NUMBER = 'number'
+    CTX_TIME = 'time'
+    CTX_SET = 'set'
+    CTX_DICT = 'dict'
+    CTX_TYPE_CHOICES = (
+        (CTX_POINT, CTX_POINT),
+        (CTX_STRING, CTX_STRING),
+        (CTX_NUMBER, CTX_NUMBER),
+        (CTX_TIME, CTX_TIME),
+        (CTX_SET, CTX_SET),
+        (CTX_DICT, CTX_DICT),
+    )
+
+    class Meta:
+        verbose_name = 'ExpressionContext'
+        verbose_name_plural = 'ExpressionContext'
```

Comment:
Why override the pluralization behavior of Django here? (Also for the `Expression` and `ExpressionType` models.)